scalyr_agent/config_main.py

```diff
@@ -1431,6 +1483,10 @@ def create_custom_dockerfile(
             # do find a need to take some action.
             sys.exit(finish_upgrade_tarball_install(paths[0], paths[1]))
 
+    if options.set_python is not None:
+        set_python_version(options.set_python)
+        print("Agent switched to {0}.".format(options.set_python))
```

Comment:
This should be part of our documentation on running agent under Python 2 / 3, but probably also doesn't hurt to it print it here as well.

If there is an agent instance running when this command is ran, it needs to be restarted for it to take an affect.

I don't think we should restart it ourselves, but just print something along the lines of "If you have an existing instance of scalyr-agent-2 process running, you need to restart it for this change to take an affect. You can do that by running /usr/init.d/scalyr-agent-2 restart...".

Reply:
Good idea. Thanks.